spec/mailers/subscription_spec.rb

```diff
@@ -35,6 +35,6 @@
   end
 
   it 'has an unsubscribe link' do
-    expect(body_lines[12]).to match(%r{http:\/\/localhost:3000\/subscriptions\/[0-9a-zA-Z\-]{418}\/unsubscribe})
+    expect(body_lines[12]).to match(%r{http:\/\/localhost:3000\/subscriptions\/[0-9a-zA-Z\-]{194}\/unsubscribe})
```

Comment:
So this is probably me being a little overzealous - `body_lines` is the body of the email split by newlines, so I'm checking that the 13th line of the email is the unsubscribe link

Reply:
Sorry, I meant the changed number `418 -> 194`.

Reply:
Aaaah, it's becauase I know the hashing algorithm gives a string length of 194 when hashing a hash with the value `{ id: somenumber }`. It probably doesn't need to be that precise TBH, hence why I had to change it when I removed data from the hash

Reply:
Aha. Good. It might be helpful to pull that out into a variable with a meaningful name (or even hash it live). I know I would be confused if that test started breaking.

Reply:
Yeah, hashing it live would be more sensible actually. I'll do that 👍